reference/promotions.v3.yml

```diff
@@ -587,8 +608,7 @@ components:
               description: |-
                 This field only has effect when the `redemption_type` is `COUPON` and `can_be_used_with_other_promotions` is `false`:
                 - When the property is set to `true`, the coupon will override the applied automatic promotions if it provides a greater discount.
-                - When the property is set to `fasle`, the coupon will not be applied if automatic promotions are already applied.
-
+                - When the property is set to `false`, the coupon will not be applied if automatic promotions are already applied.
```

Comment:
do we need to include the new field `coupon_overrides_other_promotions` in the response ? and mark this as deprecated as well ?

Reply:
yea i just found this place is same as requests. i will update the response